src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonSerializerSettingsProvider.cs

```diff
@@ -20,6 +21,11 @@ public static JsonSerializerSettings CreateSerializerSettings()
         {
             return new JsonSerializerSettings
             {
+                ContractResolver = new DefaultContractResolver
```

Comment:
Don't we have unit tests for testing default values?
It would be good to have a test (unit or functional) that explicitly test taht the default configuration defaults to camelCase